### ISSUE TITLE
Questions on FLOPs

### ISSUE THREAD

**Issue by Anonymousresults** [ZiweiWangTHU/BiDet#10]
According to your paper https://arxiv.org/pdf/2003.03961.pdf, input sized 600*1000 is employed in binarized Faster R-CNN. However, i have read your code carefully, in model BiDetResNet, the first conv layer is 'nn.Conv2d(3, first_inplanes, kernel_size=7, stride=2, padding=3, bias=False)'. Take 600*1000 input ,the FLOPs is 1,411,200,000=1345.8M. This FLOPs is calculated by torchstat. The FLOPs of the first layer single is greater than the FLOPs of whole binarized Faster R-CNN u claimed in ur paper as 781M. Please check this, thx.

**Comment by Wuziyi616** [comment]
Thank you for your interest in our work! Yes, you are right after I check my calculation when writing the paper... It seems that I used input size 300x300 as SSD when calculating Faster R-CNN. Well, that's really a mistake, the FLOPs of all Faster R-CNN models might be incorrect, but I believe they can be re-calculated by simply multiplying a scale ratio. So I think the conclusion made in our analysis (BiDet decrease FLOPs by xxx percent) is still valid.

We will modify our paper and upload it to arXiv later. We apologize for the mistake and thank you so much for pointing this out.
